admin/javascript/src/state/records/actions.js

```diff
@@ -32,7 +32,11 @@ export function fetchRecords(recordType, method, url) {
       type: ACTION_TYPES.FETCH_RECORDS_REQUEST,
       payload,
     });
-    return backend[method.toLowerCase()](populate(url, payload), {}, headers)
+    const args = method.toLowerCase() === 'get'
```

Comment:
Wouldn't fetching records always use the `GET` verb i.e. do we even need this conditional assignment?